Sources/SwiftFrameCore/Extensions/DispatchGroup+Extensions.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+extension DispatchGroup {
+
+    func ky_notifyOrExit(qos: DispatchQoS = .unspecified, flags: DispatchWorkItemFlags = [], queue: DispatchQueue, execute work: @escaping () throws -> Void) {
```

Comment:
I think I mentioned this elsewhere before: Most of the arguments seem to not be used at the call site so we could just omit them here to simplify the signature. This is ok here because it's not an API. We control all consumers so we can optimise things this way.